example/notify_inval_entry.c

```diff
@@ -275,21 +274,15 @@ static void* update_fs_loop(void *data) {
                     se->error = ret;
 
                     fuse_session_exit(se);
-                    while(!fuse_session_exited(se)) {
-                        sleep(1);
-                    }
-                    fuse_session_unmount(se);
+                    // needed to correctly exit fuse_session_loop()
+                    pthread_kill(main_thread, SIGPIPE);
```

Comment:
I don't have a strong opinion, but I wonder if this could be avoided, if we would convert to 3.12 API and then only allow 1 thread, even without "opts.singlethread".

Reply:
Yes, but let's not block on this. We've given HereThereBeDragons enough pain with this as-is, I think.